soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3355,21 +3399,19 @@ void DrawRandoEditor(bool& open) {
                     PaddedSeparator();
 
                     // Random Ganon's Trials
-                    /*
-                    ImGui::Text("Random Ganon's Trials");
-                    InsertHelpHoverText("Sets a random number or required trials to enter\nGanon's Tower.");
-                    SohImGui::EnhancementCombobox("gRandomizeGanonTrial", randoGanonsTrial, 2, 0);
-                    if (CVar_GetS32("gRandomizeGanonTrial", 0) == 0) {
-                        ImGui::PopItemWidth();
+                    ImGui::Text("Ganon's Trials");
+                    InsertHelpHoverText("Sets the number of Ganon's Trials required to dispel the barrier\n\n"
+                                        "Skip - No Trials are required and the barrier is already dispelled.\n\n"
+                                        "Set Number - Select a number of trials that will be required from the"
+                                        "slider below. Which specific trials you need to complete will be random.\n\n"
+                                        "Random Number - A Random number and set of trials will be required.");
+                    SohImGui::EnhancementCombobox("gRandomizeGanonTrial", randoGanonsTrial, 3, 0);
+                    if (CVar_GetS32("gRandomizeGanonTrial", 0) == 1) {
                         SohImGui::EnhancementSliderInt("Ganon's Trial Count: %d", "##RandoTrialCount",
-                                                       "gRandomizeGanonTrialCount", 0, 6, "", 6);
-                        InsertHelpHoverText("Set the number of trials required to enter Ganon's Tower.");
-                    RANDTODO: Switch back to slider when pre-completing some of Ganon's Trials is properly implemnted.
+                                                       "gRandomizeGanonTrialCount", 1, 6, "", 6);
+                        InsertHelpHoverText("Set the number of trials required to enter Ganon's Tower."
+                                            "The specific trials you need to complete will be randomly selected.");
```

Comment:
Don't need this since it's already explained in the main tooltip.

Reply:
Removed